src/libraries/System.Private.CoreLib/src/System/Number.Formatting.cs

```diff
@@ -1334,7 +1334,8 @@ private static unsafe void UInt32ToNumber(uint value, ref NumberBuffer number)
         {
             while (--digits >= 0 || value != 0)
             {
-                value = Math.DivRem(value, 10, out uint remainder);
+                uint remainder;
+                (value, remainder) = Math.DivRem(value, 10);
```

Comment:
Before merging, can we validate there's no unexpected regression induced by these changes?

Reply:
I tested various patterns including LICM-ability and didn't notice any differences.

Reply:
Actually there was a difference so I removed my changes for https://github.com/dotnet/runtime/pull/45074/commits/60a8d989a8222d144b298b30ae60cc507427e119

Inliner didn't want to inline the new byte version in that code + some redundant moves, will investigate and file an issue.